data/example/4-hetnet-tags.tsv

```diff
@@ -0,0 +1,22 @@
+Document	Type	ID	Offset	End
```

Comment:
Nice!

If it doesn't break too much stuff, I'd consider renaming columns like: `pubmed_id`, `kind`, `identifier`, `offset`, `end`.